grr/proto/grr_response_proto/api/osquery.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto2";
+
+package grr;
+
+import "grr_response_proto/semantic.proto";
+
+message ApiGetOsqueryResultsArgs {
+  enum Format {
+    CSV = 0;
```

Comment:
The default ("0") value of an enum has to be always named "UNKNOWN", "UNDEFINED", "DEFAULT" or alike.